spec/controllers/application_controller/timezone_spec.rb

```diff
@@ -33,7 +33,7 @@
 
       context "without a timezone" do
         it "with a system default" do
-          user = FactoryGirl.create(:user)
+          user = FactoryGirl.create(:user, :settings => {})
```

Comment:
I'm not sure if it is valid to have a user without settings.
Will `:user_with_group` fix this?